spec/models/cat_spec.rb

```diff
@@ -1,5 +1,27 @@
 require 'rails_helper'
 
 RSpec.describe Cat, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+  it 'should validate name' do
+    cat = Cat.create(age: 4, enjoys:'eating healthy food')
+    expect(cat.errors[:name]).to_not be_empty
+  end
+
+  it 'should validate age' do
+    cat = Cat.create(name: 'London', enjoys:'eating healthy food')
+    expect(cat.errors[:age]).to_not be_empty
+  end
+
+  it 'should validate enjoys' do
+    cat = Cat.create(name: 'Carrot', age: 6)
+    expect(cat.errors[:enjoys]).to_not be_empty
+  end
+
+  it 'ensures that the enjoys value is at least 10 characters' do
+    cat = Cat.create(name: 'Vanilla', age: 6, enjoys: 'bob')
+    expect(cat.errors[:enjoys]).to_not be_empty
```

Comment:
to strengthen this test, you can assert on the message you get from the error. The reason why you'd want to do this is because this test will pass when you don't provide a value for `enjoys` because the error present is actually for validating the presence of `enjoys` not actually that it's too short. Best practice for writing tests is that they assert on specifically the description provided in your it-clause (e.g. ensures that the enjoys value is at least 10 characters)